cocktails9/cocktails9/Model/ResponseList.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
```

Comment:
Consider renaming properties to make it more clear what they are expected to hold